src/dataflow/src/decode/mod.rs

```diff
@@ -404,6 +404,25 @@ fn get_decoder(
     }
 }
 
+fn try_decode(
+    decoder: &mut DataDecoder,
+    value: Option<&Vec<u8>>,
+    upstream_time: Option<i64>,
```

Comment:
Nit: Include time unit in Parameter name, and use the same variable name below.

Reply:
this was just some code golfing to try and get a particular line to not split after rustfmt. This field is going to be deleted very soon from the decoding pipeline though so I'll take care of it then